nextcloud_scan.py

```diff
@@ -7,13 +7,12 @@
 import sys
 
 class NCScan:
-    
-    API_BASE_URI = 'https://scan.nextcloud.com/api'
 
+    API_BASE_URI = 'https://scan.nextcloud.com/api'
     def __init__(self, uri, requeueMinutes=1440):
         self.uri = uri
         self.uuid = None
-        self.requeueDelta=datetime.timedelta(minutes=requeueMinutes)
+        self.requeueDelta = datetime.timedelta(minutes=requeueMinutes)
```

Comment:
See my comment below about reverting whitespace-only changes, **but** keep this one. You're right; the `=` spacing is inconsistent.